src/db/UploadClass.php

```diff
@@ -0,0 +1,121 @@
+<?php
+require_once ($root.'/DBConfig.php');
+class Upload{
```

Comment:
Needs a namespace. You can probably add it to the SqlDb namespace

Reply:
Once added please update the uploadtoDbTest file. Ensure that tests pass before merging.